static/js/settings_account.js

```diff
@@ -440,8 +440,16 @@ exports.set_up = function () {
                     window.location.href = "/login";
                 },
                 error: function (xhr) {
+                    var error_msg = "Error deactivating user: ";
+                    if (xhr.responseJSON.code === "CANNOT_DEACTIVATE_LAST_USER") {
+                        if (xhr.responseJSON.is_last_admin) {
+                            error_msg += "Cannot deactivate the only organization administrator.";
+                        } else {
+                            error_msg += "Cannot deactivate the only user. You can deactivate the whole organization though in your <a class=\"alert-link\" href=\"/#organization/organization-profile\">Organization profile settings</a>.";
+                        }
+                    }
                     $("#deactivate_self_modal").modal("hide");
-                    ui_report.error(i18n.t("Error deactivating account"), xhr, $('#account-settings-status').expectOne());
+                    $("#account-settings-status").addClass("alert-error").html(i18n.t(error_msg)).show();
```

Comment:
You can't translate a computed string with i18n.t.  You need to wrap the individual strings with `i18n.t`.  I would consider just simplifying like this:

~~~ js
if (....) {
    error_msg = i18n.t("Error: Cannot deactivate the only organazation administrator");
} else {
   error_msg = i18n.t("Error: ...")
}

Reply:
It's also worth noting that an error coming from the server will already be translated, so it's likely the correct behavior would be to not do this at all.